frontend/dockerfile/dockerfile_test.go

```diff
@@ -1122,19 +1135,29 @@ RUN [ "$(stat -c "%U %G" /mydir)" == "user user" ]
 }
 
 func testOutOfOrderStage(t *testing.T, sb integration.Sandbox) {
-	integration.SkipOnPlatform(t, "windows")
 	f := getFrontend(t, sb)
+	dockerfile := integration.UnixOrWindows(
+		`
+	FROM busybox AS target
```

Comment:
nit: can be consistent with the indentation here, these should start from col. 0.